doc/03_reference/coverage_plan.rst

```diff
@@ -202,9 +206,13 @@ Furthermore they can all occur together and must be appropriately prioritised (c
 * Exception from instruction fetch error (covered by the **FetchError** instruction category).
 * ``pmp_iside_mode_cross`` - Exception from instruction PMP violation.
 * Exception from illegal instruction (covered by the illegal instruction categories).
-* Exception from memory fetch error.
+* ``cp_ls_error_exception`` - Exception from memory fetch error.
 * ``pmp_dside_mode_cross`` - Exception from memory access PMP violation.
-* Unaligned access cases (both accesses saw error, first or second only saw error, or neither saw error) for both kinds of memory exceptions.
+* Unaligned memory access
+
+    * Cover all error and no error scenarios for memory fetch error; first access saw error, second
+      access saw error, neither access saw error
```

Comment:
I believe this is the sole remaining coverpoint to implement, rather similar to `misaligned_lsu_access_cross` but instead of PMP errors we're looking at memory fetch errors.